src/Types.ts

```diff
@@ -1,63 +1,101 @@
-import * as _ from "lodash";
-import { isNumeric } from "./utils";
+import * as _ from 'lodash'
+import { isNumeric } from './utils'
 
 // Options to be used to configure the file creator instance as a whole
 export interface GlobalOptions {
-  rowEnd: string;
-  encoding?: BufferEncoding;
-  mode?: number;
-  flag?: string;
+  rowEnd: string
```

Comment:
Not sure if this is a setting we can adjust, but preferred typescript syntax has semi-colons at the end of interface lines. See the [intro tutorial](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html) for syntax examples.

While it does appear to compile like this, I would strongly prefer to stick with Typescript standard syntax if possible.

Reply:
there are many schools of thought with respect to this point, those who find it more correct with semicolons, those who say they are more productive without... like [here](https://medium.com/@eugenkiss/dont-use-semicolons-in-typescript-474ccfe4bdb3)